crates/env/Cargo.toml

```diff
@@ -79,3 +79,13 @@ ink-debug = []
 
 # Disable the ink! provided global memory allocator.
 no-allocator = ["ink_allocator/no-allocator"]
+
+# Configurable sizes of the static buffer
+2GB-buffer = []
```

Comment:
It says that memory is to huge:
<img width="782" alt="image" src="https://github.com/paritytech/ink/assets/18346821/15b0a3b0-8928-45d3-ba3a-8e1df4ddaec2">